plugin/audit_log/tests/mtr/audit_log_default_db.test

```diff
@@ -31,6 +31,10 @@ GRANT ALL PRIVILEGES ON ąžąžąžą.* TO 'user2'@'%';
 
 # truncate audit log
 UNINSTALL PLUGIN audit_log;
+# disconnect required after https://github.com/mysql/mysql-server/commit/6a893b8b
+--source include/disconnect_connections.inc
+set names utf8;
```

Comment:
I believe this is necessary because original `default` connection before disconnect also used to have `set names utf8;`, right?

Reply:
You are right